torch/autograd/anomaly_mode.py

```diff
@@ -74,6 +74,7 @@ class detect_anomaly:
     """
 
     def __init__(self, check_nan=True) -> None:
+        """Init."""
```

Comment:
Hmmm this doesn't seem too useful lol, would prefer just to do some kind of way to skip the check if possible.

Reply:
Yeah I agree with this!

Reply:
I believe to skip it should be `# noqa:  D412`

Reply:
@soulitzer @jbschlosser I agree too but it was purely added just to satisfy the docstring requirement from `pydocstyle` as I didn't find anyway to skip it. There is a general way to ignore checking for certain error code using `--ignore` flag when we run `pydocstyle` tool but that won't help much. I like the suggestion from @svekars and I will make changes accordingly. Thanks all for the review comments!!

Reply:
I have made updates. @svekars for the `__init__` it required to use error code D107 so I have added `# noqa: D107` and verified that it worked. Thanks so much again!

Reply:
Oh, I just noticed that I accidentally committed a submodule changes as well :( I will need to fix it. Thanks!